### PR TITLE
Eat training

### PR DESCRIPTION
**Pull request by raphaelschwinger**
Please review #216 first, as these changes are included here!

This PR:
- fixes loading of model checkpoints for eat
- adds configs for training and inferencing eat models
- bash scripts for fast automating validation and evaluation

### CODE REVIEW

File: birdset/datamodule/components/transforms.py [hunk 1]
```diff
@@ -445,7 +445,7 @@ def _waveform_scaling(self, audio_augmented, attention_mask):
         return audio_augmented
    
     def _prepare_call(self):
-        if self.mode in ("valid", "test", "predict"):
+        if self.mode in ("test", "predict"):#, "valid"): ##!! important
```
Comment (lurauch) [inline birdset/datamodule/components/transforms.py:448]: valid, test, predict should be the default setting.

File: birdset/evaluate_pretrained_model.sh [hunk 2]
```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
```
Comment (lurauch) [inline birdset/evaluate_pretrained_model.sh:1]: move this to /ressources?

File: birdset/validate.sh [hunk 3]
```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
```
Comment (lurauch) [inline birdset/validate.sh:1]: ressources

File: configs/experiment/birdset_neurips24/HSN/DT/efficientnet.yaml [hunk 4]
```diff
@@ -8,12 +9,12 @@ defaults:
   - override /datamodule/transforms: bird_default_multilabel.yaml
   - override /paths: default.yaml
   - override /hydra: default.yaml
-
```
Comment (lurauch) [inline configs/experiment/birdset_neurips24/HSN/DT/efficientnet.yaml:11]: tags should only by ["birdsetDT"]? why did you change it?

File: configs/experiment/birdset_neurips24/HSN/LT/eat.yaml [hunk 5]
```diff
@@ -0,0 +1,56 @@
...
+    mode: online
+    # version: pretrain_xcl_eat_${seed}_${start_time} #to resume train run # needs to be tested
+    #version: pretraineatxcm_2 #to resume train run # needs to be tested
+
+module:
+  network:
+    torch_compile: False
+    model:
+      #local_checkpoint: /home/lrauch/mnt/train/runs/XCM/ast/2024-04-03_133040/callback_checkpoints/last.ckpt
+      local_checkpoint : /workspace/logs/train/runs/XCL/eat/2024-05-10_084240/callback_checkpoints/checkpoint-05.ckpt
```
Comment (lurauch) [inline configs/experiment/birdset_neurips24/HSN/LT/eat.yaml:34]: should be none with a comment that it can be changed

File: configs/experiment/birdset_neurips24/NES/DT/eat.yaml [hunk 6]
```diff
@@ -0,0 +1,81 @@
...
+defaults:
+  - override /datamodule: NES.yaml
+  - override /module: multilabel.yaml
+  - override /module/network: eat_soundnet.yaml
+  - override /callbacks: default.yaml
+  - override /trainer: single_gpu.yaml
+  - override /datamodule/transforms: bird_default_multilabel.yaml
+  - override /paths: default.yaml
+  - override /hydra: default.yaml
+tags: ["subset_NES", "eat", "multilabel", "5event"]
```
Comment (lurauch) [inline configs/experiment/birdset_neurips24/NES/DT/eat.yaml:12]: change tags to the tags we recently used  pls

File: notebooks/playground/rename_wandb_group.ipynb [hunk 7]
```diff
@@ -0,0 +1,73 @@
+{
```
Comment (lurauch) [inline notebooks/playground/rename_wandb_group.ipynb:1]: could you you please remove the notebooks and keep them local for now?

**Review by lurauch — CHANGES_REQUESTED** [review]

File: birdset/datamodule/components/transforms.py (hunk 1 above)
Comment (raphaelschwinger) [reply birdset/datamodule/components/transforms.py:448]:
```suggestion
        if self.mode in ("test", "predict", "valid"):
```

Comment (raphaelschwinger) [reply birdset/datamodule/components/transforms.py:448]:
```suggestion
        if self.mode in ("test", "predict", "valid"):
```

File: notebooks/playground/rename_wandb_group.ipynb (hunk 7 above)
Comment (raphaelschwinger) [reply notebooks/playground/rename_wandb_group.ipynb:1]: I would like to keep in git, then we dont have to copy it from trams again.

File: configs/experiment/birdset_neurips24/HSN/DT/efficientnet.yaml (hunk 4 above)
Comment (raphaelschwinger) [reply configs/experiment/birdset_neurips24/HSN/DT/efficientnet.yaml:11]: @lurauch I like more tags better. I guess it is not relevant to change the tags.
